system/Cookie/Cookie.php

```diff
@@ -17,6 +17,7 @@
 use DateTimeInterface;
 use InvalidArgumentException;
 use LogicException;
+use ReturnTypeWillChange;
```

Comment:
Can we really `use` a non-existent entity (on some PHP versions)?

Reply:
Yes. The class resolution will occur on the actual usage, like `new X`